content/docs/user_guide.md

```diff
@@ -514,26 +514,28 @@ ks delete ${KF_ENV} -c ${MNIST_JOB_NAME}
 
 ## Advanced Customization
 
+### Persistent Disks
+
 * Often times data scientists require a POSIX compliant filesystem
```

Comment:
Can you please also reformat the first couple of bullets before line 524 to be a simple paragraph, rather than a bulleted list.

Reply:
Yep, that felt weird to me too. ;-)